client/components/Reviews/Form.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../styles/Button.styled';
+import StyledForm from './styles/Form.styled';
+import {
+  StarRating, Recommend, Name, Email, Body, Characteristics,
+} from './FormFields';
+
+const ReviewsForm = () => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    // console.log(e.target.rating.value);
+    // console.log(e.target.nickname.value);
+    // console.log(e.target.email.value);
+    // console.log(e.target.body.value);
+    // console.log(e.target.recommend.value);
+    // console.log(e.target.size.value);
+    // console.log(e.target.width.value);
+    // console.log(e.target.comfort.value);
+    // console.log(e.target.quality.value);
+    // console.log(e.target.length.value);
+    // console.log(e.target.fit.value);
```

Comment:
Consider removing the console.log if no longer needed, no need to change right away.

Reply:
Will do! Planning on this after implementing form submit functionality.